Documentation/WritingDocForExtension/Migrate.rst

```diff
@@ -32,32 +32,222 @@ a migration tool. This tool can be used to automatically create a
 :file:`Documentation/guides.xml` from the information contained in your
 :file:`Documentation/Settings.cfg`.
 
-`Docker <https://docs.docker.com/install/>`__ needs to be installed on your
+`Docker <https://docs.docker.com/install/>`__ (or a drop-in replacement like
+`Podman <https://podman.io>`__) needs to be installed on your
 operating system for the tool to work:
 
-..  code-block:: shell
+..  tabs::
+
+    ..  group-tab:: Docker
+
+        ..  code-block:: bash
+
+            docker run --rm --pull always \
+              -v $(pwd):/project \
+              -it ghcr.io/typo3-documentation/render-guides:latest \
+              migrate Documentation
+
+    .. group-tab:: Podman
 
-    docker run --rm --pull always \
-        -v $(pwd):/project \
-        -it ghcr.io/typo3-documentation/render-guides:latest \
-        migrate Documentation
+        ..  code-block:: bash
+
+            podman run --rm --pull always \
+              -v $(pwd):/project \
+              -it ghcr.io/typo3-documentation/render-guides:latest \
+              migrate Documentation
+
+
+The last parameter (:bash:`Documentation`) is the name of the directory, where your
+:file:`Settings.cfg` is currently placed in.
 
 Try out the rendering locally
 =============================
 
 Use our Docker container to render your documentation locally. Read more about
 :ref:`local rendering <rendering-docs>`.
 
-Improve your documentation to render without warning
-====================================================
+..  _migrate-detailed-steps:
+
+Further steps to adapt to the new rendering
+===========================================
 
-If you believe you found a bug in the new PHP-based rendering, please open
+You should perform the following tasks to conclude the migration to the
+new rendering tool:
+
+1. Improve your documentation to render without warning
+-------------------------------------------------------
+
+Rendering your documentation should not yield any warnings or errors.
+
+If you get error messages, often they refer to wrong indentation, missing
+interlinks, orphaned files or outdated ReST-identifiers.
+
+If you are unable to address a warning/error with changes in your documentation
+and if you believe you found a bug in the new PHP-based rendering, please open
 an `Issue on GitHub <https://github.com/TYPO3-Documentation/render-guides/issues>`__.
 
-Recommended: Activate automatic testing
-=======================================
+2. Remove outdated files
+-----------------------------------
+
+After you have created the :file:`guides.xml` file, you can remove the old
+:file:`Settings.cfg` file.
+
+You can also delete the files :file:`genindex.rst` which was previously used
+to generate an index.
+
+The :file:`screenshots.json` is also no longer evaluated, so you can remove
```

Comment:
That is another thing, that has nothing to do with the switch. The file configured a screenshot tool which is not running anymore with TYPO3 v12. And I doubt that many users use it in their extension. AFAIK only EXT:news used it.

Reply:
But that still makes this paragraph applicable then, we could still mention it for a "new render spring cleaning"?

Reply:
I think the screenshots.json was never wide spread. I would ommit mentioning it

Reply:
Then the sentence should be adjusted. The screenshots.json has nothing to do with the docs rendering ("also no longer evaluated" - this is not true, if you are on v11 and use the testing framework v7 to create your screenshots -> this works).

Reply:
It's removed now